src/shared/main/tv/codely/shared/domain/EmailValueObject.java

```diff
@@ -0,0 +1,47 @@
+package tv.codely.shared.domain;
+
+import org.hibernate.validator.internal.constraintvalidators.hv.EmailValidator;
+
+import java.util.Objects;
+
+public abstract class EmailValueObject {
+    private static final EmailValidator emailValidator = new EmailValidator();
```

Comment:
Is this acceptable? I think a better approach would be to create an `EmailValidator` interface to not contaminate our domain with this dependency, but I tried to do it and I didn't manage to inject the implementation into this abstract class. Is it possible to do it?